jquery.pjax.js

```diff
@@ -576,7 +576,6 @@ function stripParam(url, name) {
   return url
     .replace(new RegExp('[?&]' + name + '=[^&#]*'), '')
     .replace(/[?&]($|#)/, '\1')
-    .replace(/[?&]/, '?')
```

Comment:
This line was necessary. I don't know why removing it didn't break any tests. It's here for this case:

1. URL is `foo?_pjax=xy&bar=baz`
2. `_pjax` gets stripped: `foo&bar=baz`
3. Then I transform the first `?` or `&` character into `?`, to handle the string that was result of step (2): `foo?bar=baz`